server/src/main/java/org/opensearch/search/internal/SearchContext.java

```diff
@@ -555,6 +555,14 @@ public int cardinalityAggregationPruningThreshold() {
         return 0;
     }
 
+    public boolean cardinalityAggregationHybridCollectorEnabled() {
+        return false;
+    }
+
+    public long cardinalityAggregationHybridCollectorMemoryThreshold() {
+        return Runtime.getRuntime().maxMemory() / 100; // 1% of available memory default
+    }
```

Comment:
nit: method names are quite a mouthful, though could not think of anything better myself!